### DISCUSSION TITLE
Running animation from the code behind

### DISCUSSION THREAD

**Discussion by Maruhl** [AvaloniaUI/Avalonia#17641] [Q&A] [answered]
Hi, I am trying to create a simple animation according to the documentation. However, “x:SetterTargetType” does not exist.

![grafik](https://github.com/user-attachments/assets/9d8aa3d2-8ec1-4fde-a28a-8815a33814a1)

Is there an alternative or is the [Documentation](https://docs.avaloniaui.net/docs/guides/graphics-and-animation/keyframe-animations#running-animation-from-the-code-behind) outdated? 

Avalonia 11.2.2 and VisualStudio 17.12.2 is used.

Thanks.

**Comment by stevemonaco** [comment, marked as answer]
I have no issue here on Avalonia 11.2.2 and VS 17.12.0. I suspect IntelliSense / AvaloniaVS having stale info.

First, when these issues pop up, build and check the Output window's Build in VS. Occasionally, you get "ghost errors" in XAML development, but the build output will always be correct.

If that's the issue, then close VS and delete `.vs` and bin/obj folders. Then reopen and rebuild.

**Comment by Maruhl** [comment]
Thanks, so i am running Resharper in visual studio too. And if i deactivate Resharper, it works.
